Improve API Docs (#177)
ballerina-platform/module-ballerina-cache: README.md

> The cache library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.

Reviewer: ```suggestion
The `cache` library is one of the standard library packages of the<a target="_blank" href="https://ballerina.io/"> Ballerina</a> language.
```

This package provides APIs for in-memory caching by using a semi-persistent mapping from keys to values. It consists of a default implementation based on the LRU eviction algorithm.

> For more information on all the operations supported by the `cache`, which include the below, go to [The Cache Package](https://ballerina.io/learn/api-docs/ballerina/cache/).

Reviewer: ```suggestion
For more information on all the operations supported by this package including the below, go to the [`cache` package](https://ballerina.io/learn/api-docs/ballerina/cache/).
```
